### ISSUE TITLE
VM is not updated

### ISSUE THREAD

**Issue by lbartoletti** [QGEP/docs#53]
Found for QWAT, and now for QGEP. Datamodels are both old.

**Comment by ponceta** [comment]
@sjib Is there a process or someone dedicaced for the VM update?

As far as I know, a nice dump of datamodel and demo data and an up to date qgs project is already nice.

Of course having it up to date on a VM is even better.

**Comment by m-kuhn** [comment]
@tudorbarascu you produced these if I am not mistaken. Any plans to continue this effort?

**Comment by m-kuhn** [comment]
Yep, a QGIS 3 project is just around the corner https://github.com/QGEP/QGEP/pull/468

**Comment by lbartoletti** [comment]
If it is a requirement to have a VM containing QWAT and QGEP with the latest versions, we can study the automatic creation of this one.

cc @haubourg @ponceta

**Comment by ponceta** [comment]
Automation? Really a need? I would better go the good old way on major releases only. (Except if it is now a simple procedure to create and update WM)

PUM and GIT subversion are IMHO already great tools to help minimizing update times on that side.

**Comment by ponceta** [comment]
@lbartoletti @m-kuhn @tudorbarascu If you work on this in the next days in 's-Hertogenbosch, it would be nice to list here the choices you make for these VM.

As we discussed lately, it would be nice to have such an update yearly on the QGIS LTR version, with both projects updated.

Target versions are therefore : 

- QGIS 3.10.3 (LTR)
- QGEP 1.4.0
- QWAT 1.3.4
- qgep plugin 1.2.1
- pg history viewer plugin 2.2

**Comment by sjib** [comment]
@ponceta Can you add this to the sustainability project if we still want to maintain a VM?

**Comment by ponceta** [comment]
I think there's a decision that refers more to the strategy than the technical aspects.

What we do actually want :
- An image to run CI tests and ensure stability
- An easy way to deploy a demo version of any TEKSI module (datamodel tool, packaging)
- A web based demo version of what we can expect when the module is integrated in a complete Infrastructure of GeoData (view only)
